config/packages/dev/libero_page.yaml

```diff
@@ -2,6 +2,7 @@ libero_page:
     pages:
         homepage:
             path: '/'
+            primary_listing: 'scholarly-articles/items'
```

Comment:
seems flexible enough to allow the different services, does it need additions to `sample-configuration` then?

Reply:
https://github.com/libero/browser/blob/3bb139182d3978a26d32ea23e2deffe2bb895585/vendor-extra/LiberoPageBundle/src/DependencyInjection/LiberoPageConfiguration.php#L64-L66 Is required, but can point at either content or search API (anything that returns a `libero:item-list`).